src/Features/LanguageServer/Protocol/Features/Options/IdeAnalyzerOptionsStorage.cs

```diff
@@ -28,10 +28,12 @@ public static IdeAnalyzerOptions GetIdeAnalyzerOptions(this IGlobalOptionService
             FadeOutUnusedImports = globalOptions.GetOption(FadeOutUnusedImports, language),
             FadeOutUnreachableCode = globalOptions.GetOption(FadeOutUnreachableCode, language),
             FadeOutComplexObjectInitialization = globalOptions.GetOption(FadeOutComplexObjectInitialization, language),
+            FadeOutComplexCollectionInitialization = globalOptions.GetOption(FadeOutComplexCollectionInitialization, language),
```

Comment:
There is a missing change in `AbstractUseCollectionInitializerDiagnosticAnalyzer` to actually use this option.

Reply:
Fixed, but it seems we don't have any tests :( Filed https://github.com/dotnet/roslyn/issues/61992